src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,17 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Size;
+import mate.academy.spring.validator.Email;
+import mate.academy.spring.validator.RepeatPassword;
+
+@RepeatPassword
 public class UserRequestDto {
+    @Email
     private String email;
+    @Size(min = 8)
     private String password;
+    @Size(min = 8)
```

Comment:
I think only one validation for the password is enough because you will get two repeated messages about the size of the password